docs/src/add-services/influxdb.md

```diff
@@ -47,17 +47,24 @@ if (getenv('PLATFORM_RELATIONSHIPS')) {
 
 ## Exporting data
 
-InfluxDB includes its own [export mechanism](https://docs.influxdata.com/influxdb/v1.2/tools/influx_inspect/).
-To gain access to the server from your local machine open an SSH tunnel with the Platform.sh CLI:
+To export your data from InfluxDB, follow these steps:
 
-```bash
-platform tunnel:single --relationship {{< variable "RELATIONSHIP_NAME" >}}
-```
+1. Install and set up the [`influx` CLI](https://docs.influxdata.com/influxdb/cloud/tools/influx-cli/).
+2. Connect to your InfluxDB service with the [Platform.sh CLI](../administration/cli/_index.md):
 
-By default, this opens a tunnel at `127.0.0.1:30000`.
+   ```bash
+   platform tunnel:open --host {{<variable "RELATIONSHIP_NAME" >}}
+   ```
 
-Then run InfluxDB's export commands as desired.
+   That command opens an SSH tunnel to all services on your current environment and produces output like:
 
-```bash
-influx_inspect export -compress
-```
+   ```bash
+   SSH tunnel opened to influxtimedb at: http://admin:<HASH>@127.0.0.1:30000
```

Comment:
:hammer_and_wrench: Not sure what the `<HASH>` was intended to represent, but this example seems clearer and comes from the one we have running.
```suggestion
   SSH tunnel opened to influxdb at: http://127.0.0.1:30000
```